grant_access/gpu-runner/init.txt

```diff
@@ -0,0 +1,5 @@
+pytorch-cpu
+tensorflow
```

Comment:
```suggestion
```

Let's start with feedstocks `Core` handles for now. We still need to roll out the TOS.